server/fixity.go

```diff
@@ -295,7 +295,7 @@ func itemValidate(param string) (string, error) {
 
 func statusValidate(param string) (string, error) {
 	switch param {
-	case "scheduled", "error", "mismatches", "":
+	case "scheduled", "error", "mismatches", "ok", "":
```

Comment:
and change `mismatches` to `mismatch`.

see: https://github.com/ndlib/bendo/blob/776530abf60f3d0d0ab9ff30aa418082d1703ca4/server/fixity.go#L123